tmt-app/src/Components/NavBar.jsx

```diff
@@ -0,0 +1,171 @@
+    setNavMenu(false);
+  };
+  const tabs = [
+    {
+      name: 'Majors & Careers',
+      path: '/',
+    },
+    {
+      name: 'Team',
+      path: '/team',
+    },
+    {
+      name: 'Take the Quiz',
+      path: '/quiz/classes',
+    },
+  ];
+  return (
+    <AppBar
+      sx={{
+        position: 'sticky',
+        backgroundColor: 'BTMilkTea.main',
+        marginBottom: '3.5rem',
+      }}
+    >
+      <Container maxWidth="xl">
+        <Toolbar sx={{ justifyContent: 'space-between' }}>
+          <Link key="Homepage" to="/" component={RouterLink}>
+            <img src={BTLogo} alt="logo" />
+          </Link>
+          <Box
+            sx={{
+              display: {
+                xs: 'flex',
+                md: 'none',
+              },
+            }}
+          >
+            <IconButton
+              aria-label="options"
+              color="black"
+              onClick={handleOpenNavMenu}
+            >
+              <MenuIcon />
+            </IconButton>
+            <Drawer
+              anchor="right"
+              open={navMenu}
+              onClose={handleCloseNavMenu}
+              ModalProps={{ disableScrollLock: 'true' }}
+              sx={{
+                display: {
+                  xs: 'flex',
+                  md: 'none',
+                },
+              }}
+            >
+              <Box>
+                <ListItem sx={{ justifyContent: 'flex-end' }}>
+                  <IconButton
+                    aria-label="options"
+                    color="black"
+                    onClick={handleCloseNavMenu}
+                  >
+                    <CloseIcon />
+                  </IconButton>
+                </ListItem>
+                {tabs.map((tab, id) => {
+                  return (
+                    <List key={`${tab}-{id}`}>
+                      <ListItem disablePadding onClick={handleCloseNavMenu}>
+                        <ListItemButton>
+                          <Link
+                            key={id}
+                            to={tab.path}
+                            component={RouterLink}
+                            underline="none"
+                            sx={{
+                              margin: '0 1rem',
+                              color: 'Boba.main',
+                            }}
+                          >
+                            <Typography variant="h5">{tab.name}</Typography>
+                          </Link>
+                        </ListItemButton>
+                      </ListItem>
+                    </List>
+                  );
+                })}
+              </Box>
+            </Drawer>
+          </Box>
+          <Box
+            sx={{
+              flexGrow: 1,
+              display: { xs: 'none', md: 'flex' },
+              justifyContent: 'flex-end',
+              alignItems: 'center',
+            }}
+          >
+            {tabs.map((tab, id) => {
+              return id === tabs.length - 1 ? (
```

Comment:
i think the CTA take the quiz is sort of its own action button, and doesn't quite fit in to the concept of tabs here.

Does design intend for the nav bar to have any type of active indicator? 

https://mui.com/material-ui/react-tabs/ 

maybe consider some pre-built functionality here too

Reply:
I will check with the team for any active indicator

Reply:
@jwu910 @Raymond0T Design says to just keep it as is.

Reply:
Can we spawn a new ticket and raise the accessibility issue and backlog it then?

Visual indicator for active page I thought was pretty standard accessible design